joomla/joomla-cms
# Add component (before and after) to debugger Profile Information

#### Description

This PR adds component to debugger `Profile Information` table.
It has two lines in the debugger `beforeRenderComponent` and `afterRenderComponent` just like exists for the modules.

Before:
![image](https://cloud.githubusercontent.com/assets/9630530/12249759/b8d5a068-b8b9-11e5-9156-1f383e6bffd3.png)

After:
![image](https://cloud.githubusercontent.com/assets/9630530/12249795/f2eaf708-b8b9-11e5-8089-904199e1dbe1.png)
#### How to test
1. Apply this patch.
2. Turn on the debug in global config and the debug system plugin.
3. Go to frontend and check if the two lines are in the `Profile Information` table.
